mobile/src/Modal.js

```diff
@@ -123,6 +123,13 @@ export default class CustomModal extends Component {
       }
     }
 
+    updateQuestion = (question) => {
+      this.setState({question})
+      if (this.props.showError === "red") {
```

Comment:
`showError` seems like a prop that should be a boolean?  Then render appropriate colors as an implementation detail?